src/components/message/message.scss

```diff
@@ -0,0 +1,16 @@
+.message {
+  background-color: white;
```

Comment:
Create file `/src/styles/_colors.scss`. Move all colors to it.

Reply:
done